python/rikai/viz.py

```diff
@@ -79,6 +79,12 @@ def __repr__(self):
         first_layer = self.layers[0] if self.layers else "N/A"
         return f"Draw({first_layer})"
 
+    def _repr_mimebundle_(self, include=None, exclude=None):
```

Comment:
oic for display you've converted everything into an embedded repr for databricks notebooks so we have to implement this